bundles/org.openhab.ui/web/src/pages/settings/things/inbox/inbox-list.vue

```diff
@@ -452,6 +458,15 @@ export default {
         console.error(err)
         this.$f7.dialog.alert('An error occurred: ' + err)
       })
+    masterToggle() {
+        this.checkAll = !this.checkAll
+        for (let e in this.inbox) {
+          if (this.checkAll) {
+            this.selectedItems.push(this.inbox[e].thingUID)
```

Comment:
If an item is already in the selectedItems array it will be added again with this code.

Reply:
I´m a newbie in vue/javascript programming and contributing. Hopefully this explains a little bit why there are so many issues with my commit :O)
Would it be a solution to empty the selectedItems-Array before setting all items again, or is there a better way to add all items in the list only once?

Reply:
> I´m a newbie in vue/javascript programming and contributing. Hopefully this explains a little bit why there are so many issues with my commit :O)

No worries, first-time contributions are welcome, we'll walk you through this 😃

Reply:
```suggestion    
masterToggle() {
        this.checkAll = !this.checkAll
        this.selectedItems = []
        if (this.checkAll) {
          for (let e in this.inbox) {
            this.selectedItems.push(this.inbox[e].thingUID)
          }
        }
    },

```